mnist/README.md

```diff
@@ -141,10 +154,20 @@ and running distributed.
 
 ```
 KSENV=distributed
-cd ks_app
```

Comment:
still need the cd command?

Reply:
I was expecting that they will run through the document in order, and they should already be in the ks_app directory from the previous section. I guess it's possible some people may skip straight to this step, so the extra cd might be helpful. Either way is fine with me